integration-tests/project/plugins.sbt

```diff
@@ -0,0 +1,8 @@
+resolvers += Resolver.sonatypeRepo("snapshots")
+resolvers += Resolver.sonatypeRepo("releases")
+
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
+addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "1.5.1")
+addSbtPlugin("org.lyranthe.sbt" % "partial-unification" % "1.1.2")
+
+addSbtPlugin("io.stryker-mutator" % "sbt-stryker4s" % "0.2.1")
```

Comment:
To set the version to be tested, we could do something similar to scripted sbt tests here. Setting the version as a system property (`sbt stryker -Dstryker.version=$version`)